docs/csharp/tutorials/console-webapiclient.md

```diff
@@ -1,388 +1,354 @@
-to the console. Your `Main` method now looks like this:
+   ```csharp
+   using System.Collections.Generic;
+   using System.Text.Json;
+   ```
 
-```csharp
-public static async Task Main(string[] args)
-{
-    var repositories = await ProcessRepositories();
+1. Run the application.
 
-    foreach (var repo in repositories)
-        Console.WriteLine(repo.Name);
-}
-```
+   ```dotnetcli
+   dotnet run
+   ```
 
-## Reading More Information
+   The output is a list of the names of the repositories that are part of the .NET Foundation.
 
-Let's finish this by processing a few more of the properties in the JSON packet that gets sent from the
-GitHub API. You won't want to grab everything, but adding a few properties will demonstrate a few more
-features of the C# language.
+## Configure deserialization
 
-Let's start by adding a few more simple types to the `Repository` class definition. Add these properties
-to that class:
+1. In *repo.cs*, change the `name` property to `Name` and add a `[JsonPropertyName]` attribute to specify how this property appears in the JSON.
 
-```csharp
-[JsonPropertyName("description")]
-public string Description { get; set; }
+   ```csharp
+   [JsonPropertyName("name")]
+   public string Name { get; set; }
+   ```
 
-[JsonPropertyName("html_url")]
-public Uri GitHubHomeUrl { get; set; }
+1. Add the <xref:System.Text.Json.Serialization> namespace to the `using` directives:
 
-[JsonPropertyName("homepage")]
-public Uri Homepage { get; set; }
+   ```csharp
+   using System.Text.Json.Serialization;
+   ```
 
-[JsonPropertyName("watchers")]
-public int Watchers { get; set; }
-```
+1. In *Program.cs*, update the code to use the new capitalization of the `Name` property:
 
-These properties have built-in conversions from the string type (which is what the JSON packets contain) to
-the target type. The <xref:System.Uri> type may be new to you. It represents a URI, or in this case, a URL. In the case
-of the `Uri` and `int` types, if the JSON packet contains data that does not convert to the target type,
-the serialization action will throw an exception.
-
-Once you've added these, update the `Main` method to display those elements:
-
-```csharp
-foreach (var repo in repositories)
-{
-    Console.WriteLine(repo.Name);
-    Console.WriteLine(repo.Description);
-    Console.WriteLine(repo.GitHubHomeUrl);
-    Console.WriteLine(repo.Homepage);
-    Console.WriteLine(repo.Watchers);
-    Console.WriteLine();
-}
-```
+   ```csharp
+   Console.WriteLine(repo.Name);
+   ```
+
+1. Run the application.
+
+   The output is the same.
+
+## Refactor the code
 
-As a final step, let's add the information for the last push operation. This information is formatted in
-this fashion in the JSON response:
+The `ProcessRepositories` method can do the async work and return a collection of the repositories. Change that method to return `List<Repository>`, and move the code that writes the information into the `Main` method.
```

Comment:
```suggestion
The `ProcessRepositories` method does the async work and returns a collection of the repositories. Change that method to return `List<Repository>`, and move the code that writes the information into the `Main` method.
```

Reply:
Here present tense doesn't work because it's describing the rationale for the refactoring, it's not saying what the code already does.

Reply:
I agree with Tom here.